src/components/Reports/GoalCalculator/SummaryReport/MpdGoal/MpdGoalHeaderCards/MpdGoalHeaderCards.tsx

```diff
@@ -36,7 +36,13 @@ export const MpdGoalHeaderCards: React.FC<MpdGoalHeaderCardsProps> = ({
   const { t } = useTranslation();
   const theme = useTheme();
   const locale = useLocale();
-  const { overallTotal, supportRaisedPercentage } = useGoalLineItems(goal);
+  const { overallTotal, supportRaisedPercentage } = useGoalLineItems({
+    netMonthlySalary: goal.netMonthlySalary,
+    taxesPercentage: goal.taxesPercentage,
+    rothContributionPercentage: goal.rothContributionPercentage,
+    traditionalContributionPercentage: goal.traditionalContributionPercentage,
+    ministryExpensesTotal: goal.ministryExpensesTotal,
+  });
```

Comment:
Did we need this change?

```suggestion
  const { overallTotal, supportRaisedPercentage } = useGoalLineItems(goal);
```

Reply:
It wasn't needed, it was more of a preferential thing so I will change it back. If I recall, I made that decision because there were props in goal that were not being used in useGoalLineItems.